cmd/web.go

```diff
@@ -514,6 +514,9 @@ func runWeb(ctx *cli.Context) {
 		m.Get("/labels/", repo.RetrieveLabels, repo.Labels)
 		m.Get("/milestones", repo.Milestones)
 		m.Get("/branches", repo.Branches)
+		m.Get("/stars/?:index", middleware.RepoRef(), repo.Stars)
+		m.Get("/watchers/?:index", middleware.RepoRef(), repo.Watchers)
+		m.Get("/forks", middleware.RepoRef(), repo.Forks)
```

Comment:
I think that toally depends on the size of the gogs install. I would suggest to add paging but display it only for a bigger number of rows.